tests/wallet/rpc/test_wallet_rpc.py

```diff
@@ -338,6 +338,44 @@ async def test_get_balance(wallet_rpc_environment: WalletRpcTestEnvironment):
     await assert_get_balance(wallet_rpc_client, wallet_node, cat_wallet)
 
 
+@pytest.mark.asyncio
+async def test_get_farmed_amount(wallet_rpc_environment: WalletRpcTestEnvironment):
+    env = wallet_rpc_environment
+    wallet: Wallet = env.wallet_1.wallet
+    full_node_api: FullNodeSimulator = env.full_node.api
+    wallet_rpc_client = env.wallet_1.rpc_client
+    wallet_node: WalletNode = env.wallet_1.node
+    await full_node_api.farm_blocks_to_wallet(2, wallet)
+
+    result = await wallet_rpc_client.get_farmed_amount()
+
+    expected_result = {
+        "farmed_amount": 4000000000000,
+        "farmer_reward_amount": 500000000000,
+        "fee_amount": 0,
+        "last_height_farmed": 2,
+        "pool_reward_amount": 3500000000000,
+        "success": True,
+    }
+
+    assert result == expected_result
+
+    our_ph = await wallet.get_new_puzzlehash()
+    tx = await wallet.generate_signed_transaction(
+        amount=uint64(5),
+        puzzle_hash=our_ph,
+        fee=uint64(100),
+    )
+    await wallet.push_transaction(tx)
+
+    await full_node_api.wait_transaction_records_entered_mempool(records=[tx])
+    await full_node_api.farm_blocks_to_puzzlehash(count=2, farm_to=our_ph, guarantee_transaction_blocks=True)
```

Comment:
Ok, so I see why `.process_transaction_records()` won't work here.  Seems like a shortcoming though that it doesn't pass through the farming target puzzlehash.  That's for a separate PR, but would save the `count=2` kinda just-hav'ta-know-the-implementation-details stuff I think.